index.html

```diff
@@ -319,6 +321,18 @@ <h2>Glossary</h2>
 
 
 
+<a href="#links"><abbr title="Back to Alphabetical links">&#x2191;</abbr></a>
+
+<div class="letter_anchor" id="j">J</div>
+
+<p><dfn class="lint-ignore export">Jamo</dfn>. The basic unit used to form Hangul syllables, representing vowels and consonants in Korean. In addition to code points for Jamo, Unicode encodes 11,172 Hangul <em>syllables</em>. These represent combinations of Jamo as single, pre-composed characters. In practice, such syllables are the main characters in actual use. (See also: <a href="https://www.unicode.org/faq/korean.html">Unicode Korean FAQ</a>)</p>
+
+<a href="#links"><abbr title="Back to Alphabetical links">&#x2191;</abbr></a>
+
+<div class="letter_anchor" id="k">K</div>
+
+<p><dfn class="lint-ignore export">Kana</dfn>. <a href="https://unicode.org/glossary/#kana">Unicode definition</a>: <q cite="https://unicode.org/glossary/#kana">A collective term for the two syllabic scripts used (along with <a href="https://unicode.org/glossary/#kanji">kanji</a> and <a href="https://unicode.org/glossary/#romaji">romaji</a>) by the Japanese writing system. The two forms are <em><a href="https://unicode.org/glossary/#hiragana">hiragana</a></em> and <em> <a href="https://unicode.org/glossary/#katakana">katakana</a></em>.</q></p>
```

Comment:
For the unicode.org links, should we do `target="_blank"`? Those links take one away from our glossary. Ideally those should link *locally* to our glossary (and our glossary expose those terms for those who need it)?

Reply:
Currently, all unicode.org links in this document do not have `target="_blank"`. If we want to add it, I can raise a separate issue/PR to discuss this. I agree with this, FWIW.

Regarding local links, I also agree, but sometimes we don't have relevant items in our glossary, and if we have them, I will link to the local ones.